app/controllers/Application.scala

```diff
@@ -127,6 +127,18 @@ class Application(
     )
   }
 
+  def showcase(): Action[AnyContent] = CachedAction() { implicit request =>
```

Comment:
You can get rid of the `()` on this method

Reply:
done